### PR TITLE
Allow custom attributes on video helper

### PR DESCRIPTION
**Pull request by afbora**
## Describe the PR

Custom attributes can now be added to iframes for video helper. For fullscreen mode, the deprecated `allowfullscreen` attribute has been changed to the new `allow` attribute.

### Enhancements

- Custom iframe attributes allowed on video helper

## Breaking changes

None

## Related issues/ideas

None

## Ready?

- [x] Unit tests for fixed bug/feature
- [x] In-code documentation (wherever needed)
- [x] CI checks pass

## When merging

- [x] Add to [website docs release checklist](https://github.com/getkirby/getkirby.com/pulls) (if needed)
- [x] Add changes to release notes draft in Notion

### CODE REVIEW

File: config/tags.php [hunk 1]
```diff
@@ -314,6 +314,21 @@
...
+                    unset(
+                        $attrs['autoplay'],
+                        $attrs['controls'],
+                        $attrs['loop'],
+                        $attrs['muted'],
+                        $attrs['poster'],
+                        $attrs['preload']
+                    );
+                }
+
```
Comment (afbora) [inline config/tags.php]: I had to remove attributes in video kirbytag that iframe doesn't support.

Comment (bastianallgeier) [reply config/tags.php]: This is probably exactly the reason why I added the allow list instead.

Comment (afbora) [reply config/tags.php]: I guess, yes 😅

**Comment by afbora** [comment]
So what would be an option for user defined allowed attributes and defaults like in `kirbytext.video` instead of allowing everything?

**Comment by bastianallgeier** [comment]
I think the main problem here is that we could easily define an allow list based on standard attributes, but we also want to allow custom attributes like those for the UI framework. In this case we cannot really use an allow list or we simply allow any attribute with a prefix, which is pretty complicated again. 

I have to say that I'm not really happy about the massive unset.

**Comment by afbora** [comment]
What do you think splitting attributes like in https://github.com/getkirby/kirby/pull/3680/commits/2fc1866f02c8a0b3a1875a416e711a2cbeb2f0d1 ?

**Comment by bastianallgeier** [comment]
That's pretty clever!

**Review by bastianallgeier — APPROVED** [review]

**Pull request merged**